### ISSUE TITLE
osmscout-sailfish at raspberry-pi 

### ISSUE THREAD

**Issue by susnicek** [Karry/osmscout-sailfish#239]
Dear developer,
I would like to use the osmscout-sailfish in my raspberry-pi device (with raspian os). Is there any way how to compile it for this device?
Thank you for your advice in advance.
Best regards, susnicek

**Comment by Karry** [comment]
Hi.

You may build it for standard Linux distributions. So it should not be problem even for Raspbian. Here is howto: https://github.com/Karry/osmscout-sailfish/wiki/Local-development

But problem are Silica components, that are close-source partly. So, you cannot use Sailfish UI on ordinal Linux. There is limited support for standard QML components, you just need to start application with `--desktop` argument. But you should be aware that this UI is just for testing and development. It is limited just to map.

Also, you may try OSMScout2 demo from libosmscout library itself: https://github.com/Framstag/libosmscout/ This application/project is just tin layer around this library that adds UI for Sailfish OS and storage for GPS tracks.

What is target state that you want to achieve?

Regards, Lukas

**Comment by susnicek** [comment]
Hi, thank you for the advice!
I succeeded to compile the sources on my Debian.
My goal is to have a simple offline map viewer and gpx file reader, which works directly with database files and the maps are rendered during the viewing process (the rendering style is not much important for me). Maybe I am at a wrong address, and Osmscout is not capable of this.
I achieved this goal with [GPXsee](https://www.gpxsee.org/) application, whose code I slightly modified for my purposes. But the offline maps in form of MBTiles were too large (zoom 19) even for a small area.

Bests, susnicek

**Comment by susnicek** [comment]
Hi, thank you for the advice!
I succeeded to compile the sources on my Debian.
My goal is to have a simple offline map viewer and gpx file reader, which works directly with database files and the maps are rendered during the viewing process (the rendering style is not much important for me). Maybe I am at a wrong address, and Osmscout is not capable of this.
I achieved this goal with [GPXsee](https://www.gpxsee.org/) application, whose code I slightly modified for my purposes. But the offline maps in form of MBTiles were too large (zoom 19) even for a small area.

Bests, susnicek

**Comment by susnicek** [comment]
Hi, thank you for the advice!
I succeeded to compile the sources on my Debian.
My goal is to have a simple offline map viewer and gpx file reader, which works directly with database files and the maps are rendered during the viewing process (the rendering style is not much important for me). Maybe I am at a wrong address, and Osmscout is not capable of this.
I achieved this goal with [GPXsee](https://www.gpxsee.org/) application, whose code I slightly modified for my purposes. But the offline maps in form of MBTiles were too large (zoom 19) even for a small area.

Bests, susnicek
